### DISCUSSION TITLE
[Projects Beta] Allow cloning/duplicating a project

### DISCUSSION THREAD

**Discussion by ctron** [community/community#7908] [Projects and Issues] [answered]
Assume you have set up a project for release 1.0. Now you want to create/prepare a new project for release 1.1. Most likely, you want to take the existing setup, without the assigned issues.

I think it would make sense to have a "clone", "copy", "duplicate" option … which allows to duplicate a project setup, optionally dropping all issues form on, so that the new project would be empty.

**Comment by mobile-sergey** [comment]
Why you don't use [Milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones), [Branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/viewing-branches-in-your-repository), [Tags and Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/viewing-your-repositorys-releases-and-tags) for devide projects to releases 1.0 and 1.1?

**Reply by ctron** [reply]
Milestones are tied to a single repository. So are branches, tags, and releases.

**Reply by mobile-sergey** [reply]
> Milestones are tied to a single repository. So are branches, tags, and releases.

My question is: Why you use 2 repositories to 2 releases? Why you don't develop all releases on single repository?
As for me all releases may be developed in single repository (for example in different branches)

**Reply by ctron** [reply]
So maybe your case is different. We do not use two releases. But a single one. However this consists of content coming from multiple repositories. (client, server, deployment script, docs, spec, …). All of this ends up in the release of a single repository. However, remaining work must be tracked across multiple repositories.

**Reply by mobile-sergey** [reply]
You wrote on topic:
> Assume you have set up a project for release 1.0. Now you want to create/prepare a new project for release 1.1. Most likely, you want to take the existing setup, without the assigned issues.

That is why I think that you use two releases. 

If you talking about client, server, etc and you need to copy one issue to other repositories. I agree with you. And today I need this option too. We get error on server and our teamlead create issue in server repository. But it was error on mobile. And we need to copy this issue to 2 repositories: iOS and Android. And no option to copy/duplicate/etc. It was uncomfortable to create this two issues.

**Comment by mlcaraldi** [comment]
The ability to copy a project would be really useful!

**Comment by nk-rakanishu** [comment]
This is a standard feature on the non-beta projects and is incredibly useful. Please prioritize this upgrade!

**Comment by bjornai** [comment]
Im using the new beta project to create "test" issues for my editors. Would save me a lot of typing if I could save a copy as a starting point and then just create new test projects as time goes by based on that one template

**Comment by GoodBoyDigital** [comment]
I would love to see this too! New beta projects are amazing, but a bit of a pain having to rebuild our set up each time we start a new game.

**Comment by archon810** [comment]
I just spent a couple of hours creating a project and fine-tuning it for one of my repos. There are many views, fields added, etc. Now I'd like to replicate it in other repos, but without cloning, it's going to take ages (not nearly as long as the original fine-tuning, but still unnecessarily long). Please add cloning/duplicating of projects.

**Comment by archon810** [comment]
Related: https://github.com/github-community/community/discussions/14575.

**Comment by ashleyflorence** [comment]
It would be helpful to duplicate a project board as it is set up the way we need it to be for future iterations of this project type, or create a custom project layout that can be applied to new boards moving forward.

**Comment by baden-nira** [comment]
Related to https://github.com/github-community/community/discussions/8357

**Comment by rileybroughten** [comment, marked as answer]
👋 Wanted to drop in here to say that in addition to being able to [make a copy of a project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/copying-an-existing-project), project templates are now in public beta and we would love your feedback! Here is a link to the [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-your-project/managing-project-templates-in-your-organization), as well as the [discussion link](https://github.com/orgs/community/discussions/54576) for any feedback that you may have. Please let us know how it works for you and where else we can improve! 🎉
